Demo/Demo/SwiftUIComponents/SwiftUICardDemo.swift

```diff
@@ -35,12 +35,12 @@ struct SwiftUICardDemo: View {
                         cvvText = cardFormatter.formatFieldWith(newValue, field: .cvv)
                     }
                 HStack {
-                    Toggle("Should Vault", isOn: $shouldVaultSelected)
+                    Toggle("Should Vault with Purchase", isOn: $shouldVaultSelected)
+                    // TODO: turn on if vault with purchase on sample server is implemented
+                        .disabled(true)
```

Comment:
Instead of `.disabled(true)` I think it makes more sense to use `.hidden()` to remove this from the UI completely until it can be turned back on. What do you think?

Reply:
I left that there because I thought it would be strange to have optional customerID field in there by itself. 
Right now, I am using that field for testing Vault without Purchase flow with the button for Vault without Purchase right below. This is temporary while I do some refactor on the UI for demo app.

I am working on making opening screen for having two options, vault without purchase (with options for payment methods) and checkout with payment options. I think having App struct replacing SceneDelegate/AppDelegate is straight forward, I tried commenting out those files and having App file as entry point and it seems to be fine.

I want to extract out the Card view from current demo screen so I can reuse it for the Vault without Payment flow. 
I am working on moving Card.swift and Address.swift to Core module so they can be reused.

Reply:
> I left that there because I thought it would be strange to have optional customerID field in there by itself.

I think either option is fine since this is internal. To me though a disabled element typically indicates there is an action that can be done to set the state from disabled to enabled whereas hidden indicates that it cannot should be used in it's current state.

I think before we get too far into building UI it's worth tackling removing UIKit completely from the Demo app. This will make it much easier to add and update demo app features. A few of the views are still tightly coupled to UIKit views which will make it much harder to adapt the new views you want.

> I am working on moving Card.swift and Address.swift to Core module so they can be reused.

I don't think we want to move these into core unless they can be used across multiple modules. Especially card since merchants using PayPal or NXO should not ever need to interact with the card model. I may be missing some context that you and @sshropshire have already discussed though, but imo doesn't make sense to move those into core.

Reply:
I agree about removing UIKit. That's why I was investigating removing SceneDelegate/AppDelegate.

For Card.swift, for right now, it is a form that we use to build encodable structs for PaymentSource. I wanted a way to use this for Demo app for Vault without Purchase flow and GraphQL request for update setup token.  I will discuss it further with @sshropshire

Reply:
That makes sense what you are saying about PayPal and NXO though. Let me think on it and discuss with Steven.

Reply:
That sounds good! I am sure y'all will come up with something that makes sense for both platforms while keeping things DRY where possible. 😄